web/src/views/SubmissionPortal/Components/HarmonizerSidebar.vue

```diff
@@ -0,0 +1,162 @@
+<script lang="ts">
+import {
+  defineComponent, PropType, ref,
+} from '@vue/composition-api';
+import FindReplace from '@/views/SubmissionPortal/Components/FindReplace.vue';
+import type { HarmonizerApi } from '@/views/SubmissionPortal/harmonizerApi';
+import ContactCard from '@/views/SubmissionPortal/Components/ContactCard.vue';
+import ImportExportButtons from '@/views/SubmissionPortal/Components/ImportExportButtons.vue';
+import ColumnHelp from '@/views/SubmissionPortal/Components/ColumnHelp.vue';
+import MetadataSuggester from '@/views/SubmissionPortal/Components/MetadataSuggester.vue';
+import { ColumnHelpInfo, HarmonizerTemplateInfo } from '@/views/SubmissionPortal/types';
+
+/**
+ * The tabbed Data Harmonizer sidebar.
+ */
+export default defineComponent({
+  components: {
+    MetadataSuggester,
+    ColumnHelp,
+    ImportExportButtons,
+    ContactCard,
+    FindReplace,
+  },
+  props: {
+    /**
+     * Help information for the currently selected column.
+     */
+    columnHelp: {
+      type: Object as PropType<ColumnHelpInfo | null>,
+      default: null,
+    },
+    /**
+     * Information about the active template.
+     */
+    harmonizerTemplate: {
+      type: Object as PropType<HarmonizerTemplateInfo>,
+      required: true,
+    },
+    /**
+     * Whether the current user is allowed to edit metadata.
+     */
+    metadataEditingAllowed: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * The Harmonizer API instance.
+     */
+    harmonizerApi: {
+      type: Object as PropType<HarmonizerApi>,
+      required: true,
+    },
+  },
+  emits: ['export-xlsx', 'import-xlsx'],
+  setup(props, { emit }) {
+    const tabModel = ref(0);
+    const TABS = [
+      {
+        icon: 'mdi-information-outline',
+        label: 'Column Help',
+      },
+      {
+        icon: 'mdi-text-search',
+        label: 'Find & Replace',
+      },
+      {
+        icon: 'mdi-assistant',
+        label: 'Metadata Suggester',
+      },
+      {
+        icon: 'mdi-swap-vertical',
+        label: 'Import & Export',
+      },
+      {
+        icon: 'mdi-help-circle-outline',
+        label: 'Help',
+      },
+    ];
+
+    const handleImport = (...args: never[]) => {
+      emit('import-xlsx', ...args);
+    };
+
+    return {
+      TABS,
+      handleImport,
+      tabModel,
+    };
+  },
+});
+</script>
+
+<template>
+  <div class="harmonizer-sidebar-content">
+    <v-tabs
+      v-model="tabModel"
+      grow
+    >
```

Comment:
I don't think this would show up for most people's setups, but when I had the dev console open I noticed that I could scroll the tabs out of frame. Do you think it might be worth pinning them?

![image](https://github.com/user-attachments/assets/5409ed88-9ef7-464a-8ce2-b77e271b3bad)

Reply:
That's a good observation. Even without the dev console you can get into this state with some of the columns (like "broad-scale environmental context") that have a long description or in the suggester tab if you have a lot of pending suggestions.

I added some flex utility classes to ensure that the tab content is what scrolls.

Reply:
I'm still seeing this after pulling the most recent changes. I messed around a bit and it looks like [position: sticky](https://developer.mozilla.org/en-US/docs/Web/CSS/position?v=example#sticky) might do the trick. If you want to try the flexbox solution as well that's fine. Either way I don't think this should hold up the merge, we can spin this out into an issue. 

Let's get this into dev for more thorough testing.

Reply:
Yeah agreed on moving forward. Maybe you'll have to show me how you're reproducing the issue; I might have been doing something slightly different.